cmd/mattermost/commands/channel_test.go

```diff
@@ -113,3 +113,11 @@ func TestCreateChannel(t *testing.T) {
 	name = name + "-private"
 	CheckCommand(t, "channel", "create", "--display_name", name, "--team", th.BasicTeam.Name, "--private", "--name", name)
 }
+
+func TestRenameChannel(t *testing.T) {
+	th := api4.Setup().InitBasic()
+	defer th.TearDown()
+
+	channel := th.CreatePublicChannel()
+	CheckCommand(t, "channel", "rename", th.BasicTeam.Name+":"+channel.Name, "NewChannelName", "--display_name", "New Display Name")
```

Comment:
I would add here the assertions about the channel object renamed in the database too.